ERCS/erc-draft_embedded_liquidity.md

```diff
@@ -0,0 +1,404 @@
+---
```

Comment:
```suggestion
---
eip: 7649
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.